Setting ACLs with chimera command line utility fails with NoClassDefFoundError

chimera:/disk# setfacl test USER:501:+r:fo
    Exception in thread "main" java.lang.NoClassDefFoundError: org/dcache/acl/parser/ACEParser
	at org.dcache.chimera.cli.Shell$SetFaclCommand.call(Shell.java:759)
	at org.dcache.chimera.cli.Shell$SetFaclCommand.call(Shell.java:657)
	at org.dcache.util.cli.AnnotatedCommandExecutor.execute(AnnotatedCommandExecutor.java:130)
	at org.dcache.util.cli.CommandInterpreter$CommandEntry.execute(CommandInterpreter.java:260)
	at org.dcache.util.cli.CommandInterpreter.doExecute(CommandInterpreter.java:169)
	at org.dcache.util.cli.CommandInterpreter.command(CommandInterpreter.java:155)
	at org.dcache.util.cli.ShellApplication.execute(ShellApplication.java:149)
	at org.dcache.util.cli.ShellApplication.console(ShellApplication.java:191)
	at org.dcache.util.cli.ShellApplication.start(ShellApplication.java:84)
	at org.dcache.chimera.cli.Shell.main(Shell.java:83)
    Caused by: java.lang.ClassNotFoundException: org.dcache.acl.parser.ACEParser
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 10 more

--- a/skel/share/lib/loadConfig.sh
+++ b/skel/share/lib/loadConfig.sh
@@ -138,8 +138,9 @@ printLimitedClassPath() # $1..$n = list of jar files
     classpath="$(printPluginClassPath)"
     classes_path="$(getProperty dcache.paths.classes)"
     for name in "$@"; do
-        jar="$(echo $classes_path/$name-*.jar)"
-        classpath="${classpath}:${jar}"
+        for jar in $classes_path/$name-*.jar; do
+            classpath="${classpath}:${jar}"
+        done
     done
 
     echo ${classpath#:}

